package.json

```diff
@@ -9,11 +9,11 @@
     "npm": ">=1.3.0 <3.0.0"
   },
   "devDependencies": {
-    "eslint": "^8.29.0",
-    "puppeteer": "^18.1.0",
-    "grunt": "^1.5.3",
+    "eslint": "^8.34.0",
+    "puppeteer": "^19.7.0",
+    "grunt": "^1.6.1",
```

Comment:
There is a warning in GitHub actions logs:
```
npm WARN notsup Unsupported engine for grunt@1.6.1: wanted: {"node":">=16"} (current: {"node":"14.21.2","npm":"6.14.17"})
```
I think we need to bump `node-version` to use `grunt@1.6.1` :thinking:

Reply:
I think that's gone, we have a warning about puppeteer which is coming from `grunt-contrib-qunit` I think https://github.com/gruntjs/grunt-contrib-qunit/issues/194 but we also had it before